keyboards/kinesis/kint36/config.h

```diff
@@ -0,0 +1,87 @@
+/* Copyright 2020 QMK
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+/* USB Device descriptor parameter */
+#undef VENDOR_ID
+#define VENDOR_ID 0x1209
+#undef PRODUCT_ID
+#define PRODUCT_ID 0x345C
+#undef DEVICE_VER
+#define DEVICE_VER 0x0001
+#undef MANUFACTURER
+#define MANUFACTURER "https://github.com/stapelberg"
+#undef PRODUCT
+#define PRODUCT "https://github.com/kinx-project/kint"
```

Comment:
These shouldn't really be URLs...

Reply:
Huh, why not? I find URLs to be the most elegant thing to put into the product field for self-built devices. That way, it’s very clear where to find more information/sources :)

Reply:
It's meant to be the product *name*, not the URL. It would just look weird.
![image](https://user-images.githubusercontent.com/4781841/111449230-4ee3dc00-8763-11eb-8ec7-151f221d57c1.png)

Reply:
Done.